ViewModel/HyvaGrid/Navigation.php

```diff
@@ -419,4 +422,28 @@ private function sortButtonConfig(array $buttonsConfig): array
 
         return $buttonsConfig;
     }
+
+    public function getExports(): array
+    {
+        return map([$this, 'buildExport'], $this->sortExportsConfig($this->navigationConfig['exports'] ?? []));
+    }
+
+    private function buildExport(array $config): GridExportInterface
+    {
+        return $this->gridExportFactory->create(merge([], $config));
+    }
+
+    private function sortExportsConfig(array $config): array
```

Comment:
This is almost an exact copy of the `sortButtonsConfig` method.
I'm sure it can be reused instead of duplicated.